docs/shared/cloud-router-discontinuation-notice.mdx

```diff
@@ -0,0 +1,9 @@
+<Caution>
+
+Apollo is discontinuing Serverless and Dedicated plans, which use cloud routers. Cloud routers will no longer be available after [TODO SUNSET DATE].
```

Comment:
Adding a PR comment here so that we don't forget to address the [TODO SUNSET DATE]